src/app/components/deathStar.js

```diff
@@ -0,0 +1,25 @@
+export const deathStar = ({ sec }) => {
+    const deathStarDiv = document.createElement('div');
+    const image = document.createElement('img');
+    console.log(sec);
+
+    deathStarDiv.className = 'deathStarDiv';
+    image.className = 'deathStar';
+    image.src = '../static/assets/img/modes/deathStar/deathstar.png';
+    deathStarDiv.appendChild(image);
+    document.body.appendChild(deathStarDiv);
```

Comment:
Nie powinieneś dodawać elementu to `body`. Dodawaniem komponentów będzie zajmować się oddzielna funkcja.